service/src/service/postgres.go

```diff
@@ -176,6 +176,9 @@ func (s *PostgresClient) getConnection(connection QueryConnection) (*pgx.Conn, s
 	}
 
 	config, errConfig := pgx.ParseConfig(connString)
+	config.RuntimeParams = map[string]string{
+		"application_name": "Ivory",
```

Comment:
Maybe also add version of `Ivory` here

Reply:
yeah good idea [add version to application name](https://github.com/veegres/ivory/pull/455/commits/ddbde91f606e1754e2d6e3c56a0914f6bb5792c1)